config/routes.rb

```diff
@@ -383,9 +383,6 @@
       #
       # These have been left in temporarily to prevent any impact to users
```

Comment:
only thing i would change is removing this comment as well

Reply:
Yes right, thank you!